sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html

```diff
@@ -0,0 +1,77 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. -#} {% set active_page =
+"appeal-128" %} {% set page_title_text = _('Experience the future of
+Thunderbird') %} {% extends "includes/base/base.html" %} {% block page_title
+%}{{ page_title_text }}{% endblock %} {% block page_desc %}{{ _('Thunderbird is
+a free email application that’s easy to set up and customize - and it’s loaded
+with great features!') }}{% endblock %} {% block breadcrumbs %}{% endblock %} {%
+block extra_meta %}
+<meta name="robots" content="noindex,nofollow" />
+{% endblock %} {% block content %}
+<section>
+  <div class="container">
+    <section class="appeal-welcome">
+      <header>{{_('Welcome to the evolution of Supernova!')}}</header>
+      <p>
+        {{_('With revolutionary code improvements, Nebula sets the stage for
+        faster delivery of new features, reinforced stability, stellar
+        quality-of-life enhancements and visual refinements.')}}
+      </p>
+    </section>
+
+    <section class="speed">
+      <header>{{_('')}}</header>
+      <p>{{_('')}}</p>
```

Comment:
Fun fact about empty msgid, Pontoon uses it for project details/contact info 😄  https://github.com/thunderbird/thunderbird.net-l10n/blob/master/en_CA/LC_MESSAGES/messages.po#L9